media_kit/lib/src/player/web/player/real.dart

```diff
@@ -418,15 +451,7 @@ class WebPlayer extends PlatformPlayer {
         );
       }
 
-      try {
-        element.src = _playlist[_index].uri;
-      } catch (exception) {
-        // PlayerStream.error
-        final e = exception as html.DomException;
-        if (!errorController.isClosed) {
-          errorController.add(e.message ?? '');
-        }
-      }
+      loadResource(_playlist[_index].uri);
```

Comment:
Have you made sure to replace all `.src =` assignments?

Reply:
> Have you made sure to replace all `.src =` assignments?

all are replaced;